packages/experimental/AppearanceAdditions/src/NativeAppearanceAdditions.ts

```diff
@@ -1,21 +1,19 @@
-import type { AccessibilityContrastOption, SizeClass, UserInterfaceLevel } from './NativeAppearanceAdditions.types';
-
 export const NativeAppearanceAdditions = {
   // eslint-disable-next-line @typescript-eslint/no-empty-function
   addListener: (_: string) => {},
   // eslint-disable-next-line @typescript-eslint/no-empty-function
   removeListeners: (_: number) => {},
   horizontalSizeClass: () => {
     console.warn('NativeAppearanceAdditions is only available on iOS');
-    return 'regular' as SizeClass;
+    return {};
   },
   userInterfaceLevel: () => {
     console.warn('NativeAppearanceAdditions is only available on iOS');
-    return 'base' as UserInterfaceLevel;
+    return {};
   },
   accessibilityContrastOption: () => {
     console.warn('NativeAppearanceAdditions is only available on iOS');
-    return 'normal' as AccessibilityContrastOption;
+    return {};
```

Comment:
Why these changes?

Reply:
Hmm so this file is just stubs for the other platforms. These methods changed from returning strings to returning dictionaries, so I changed it to return an empty dictionary. I'll double check on macOS to make sure other platforms aren't affected

Reply:
Tested by just running the macOS tester and don't see any issues. I realized on macOS though we don't seem to support multiple windows either...at least I haven't figure out how to open another FluentTester window. Will look into what's needed to support this